serializations.md

```diff
@@ -815,7 +828,7 @@ version 2 (valid from Fortuna hardfork on)
   seen by each participant. Note that could yeld different results because of
   their different view of the chain.
 
-#### Channel
+#### Channel (version 1, until Fortuna release)
```

Comment:
So channel version 1 is not legal in Fortuna. Right?

Reply:
If it already exists it will stay version 1, but new channels will be version 2 from Fortuna height.